### DISCUSSION TITLE
Unraid user json mapping

### DISCUSSION THREAD

**Discussion by zarttak** [luigi311/JellyPlex-Watched#201] [Q&A] [answered]
I need some help with Unriad. Most of the users in plex have different usernames than they do in Jellyfin. So need some help in creating the json map for the users. Please point me in the direction on where I can learn or figure out how to create this.
![jellyplex](https://github.com/user-attachments/assets/755af051-a03c-41fb-83af-4a7a90b75c56)

**Comment by twinkie0101** [comment]
Did you ever figure this out?

**Comment by luigi311** [comment, marked as answer]
You would put in something like this
{"PlexUsername": "JellyfinUsername", "Second User": "Mapped Second User"}

Just know you cant use multiple definitions so you cant define say PlexUsername to both JellyfinUsername and EmbyUsername. It can only be one.
